app/move/controllers/create/personal-details.test.js

```diff
@@ -135,6 +135,41 @@ describe('Move controllers', function() {
       })
     })
 
+    describe('#savePerson()', function() {
+      const mockId = '1234567'
+
+      context('without id', function() {
+        beforeEach(function() {
+          sinon.spy(personService, 'create')
+          sinon.spy(personService, 'update')
+
+          controller.savePerson(undefined, personMock)
+        })
+
+        it('should create a person via the personService', function() {
+          expect(personService.create).to.be.calledOnceWithExactly(personMock)
+          expect(personService.update).to.not.be.called
```

Comment:
Could break these out into two separate assertions. Might make it easier to debug if one fails.